backend/typescript/graphql/resolvers/dashboardResolvers.ts

```diff
@@ -27,6 +27,15 @@ const dashboardResolvers = {
       );
       return applications;
     },
+    applicationsBySecondChoiceRole: async (
+      _parent: undefined,
+      { secondChoice }: { secondChoice: string },
```

Comment:
Instead of the `secondChoice` type to be string, lets make it `ApplicantRole`. You can grab this `ApplicantRole` type from the frontend file